deploy/services-demo/conf/spar.demo.yaml

```diff
@@ -0,0 +1,30 @@
+saml:
+  version:     SAML2.0
+  log_level:   Debug
+
+  sp_host:     0.0.0.0
+  sp_port:     8088
+  sp_app_uri:  http://localhost:8080/ # <--- change this to point to a reachable web app
```

Comment:
ah, right, the web-app is not included here.  i wonder how hard that would be?  :-)